src/shell/grabs/moving.rs

```diff
@@ -490,7 +490,32 @@ impl PointerGrab<State> for MoveGrab {
         event: &MotionEvent,
     ) {
         self.update_location(state, event.location);
-
+        {
+            let mut borrow = self
+                .seat
+                .user_data()
+                .get::<SeatMoveGrabState>()
+                .map(|s| s.lock().unwrap());
+            if let Some(grab_state) = borrow.as_mut().and_then(|s| s.as_mut()) {
+                // is this workspace location or output location?
+                let location =
+                    grab_state.window_offset.as_global().to_f64() + grab_state.location.as_global();
+                for (window, offset) in self.window.windows() {
+                    match window.0.underlying_surface() {
+                        WindowSurface::Wayland(toplevel) => {
+                            update_reactive_popups(
+                                toplevel,
+                                // TODO: give some thought to round vs floor
+                                (location + offset.as_global().to_f64()).to_i32_round(),
+                                self.window_outputs.iter(),
```

Comment:
The outputs are for finding the anchor point and the anchor point must be within the surface, so it will be within the same outputs as the window, right?